test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -145,13 +153,25 @@ class test_ucp_rma : public test_ucp_memheap {
         iov[iov_count - 1].length += size % iov_count;
     }
 
+    void request_param_set_memh(ucp_request_param_t *param, ucp_mem_h memh)
```

Comment:
maybe move param.op_attr_mask = 0; into this function
and rename it to request_param_init()